pcap.c

```diff
@@ -3967,19 +3952,6 @@ pcap_live_dump_ended(pcap_t *p, int sync)
 {
 	return (p->live_dump_ended_op(p, sync));
 }
-
```

Comment:
`pcap_get_airpcap_handle()` is part of the ABI, so it shouldn't be removed.  Instead, it should always set `p->errbuf` to a message that indicates the AirPcap devices aren't supported and return NULL.